backend/docker-services/openzaak/Readme.md

```diff
@@ -0,0 +1,111 @@
+# Open Zaak
+
+The `docker-compose.yml` compose file is available to run an instance of Open Zaak.
+
+## docker compose
+
+Start an instance in your local environment from this directory:
+
+```bash
+docker compose up
+```
+
+This brings up the admin at http://localhost:8003/admin/. You can log in with the `admin` / `admin`
+credentials.
+
+## Load fixtures
+
+The fixtures in `./fixtures` are automatically loaded when the Open Zaak container starts.
```

Comment:
Is this the desired behavior? I would expect this to be an explicit action? Also: i can't find the implementation?

Reply:
It is a built-in feature of Open Zaak: https://github.com/open-zaak/open-zaak/blob/6f5aaec6fed0879c96ae34c69814d8d5a139562e/bin/docker_start.sh#L25

Reply:
Ok 👍